dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -439,43 +442,77 @@ export const Card = ({
 							</AvatarContainer>
 						)}
 						{media.type === 'video' && (
-							<div
-								data-chromatic="ignore"
-								data-component="youtube-atom"
-								css={css`
-									display: block;
-									position: relative;
-									${getZIndex('card-nested-link')}
-								`}
-							>
-								<Island>
-									<YoutubeBlockComponent
-										id={media.mainMedia.elementId}
-										elementId={media.mainMedia.elementId}
-										assetId={media.mainMedia.videoId}
-										duration={media.mainMedia.duration}
-										posterImage={media.mainMedia.images}
-										overrideImage={media.imageUrl}
-										width={media.mainMedia.width}
-										height={media.mainMedia.height}
-										origin={media.mainMedia.origin}
-										mediaTitle={headlineText}
-										expired={media.mainMedia.expired}
-										format={format}
-										isMainMedia={true}
-										hideCaption={true}
-										role="inline"
-										stickyVideos={false}
-										kickerText={kickerText}
-										pauseOffscreenVideo={
-											pauseOffscreenVideo
-										}
-										showTextOverlay={
-											containerType === 'fixed/video'
-										}
-									/>
-								</Island>
-							</div>
+							<>
+								{showMainVideo ? (
+									<div
+										data-chromatic="ignore"
+										data-component="youtube-atom"
+										css={css`
+											display: block;
+											position: relative;
+											${getZIndex('card-nested-link')}
+										`}
+									>
+										<Island>
+											<YoutubeBlockComponent
+												id={media.mainMedia.elementId}
+												elementId={
+													media.mainMedia.elementId
+												}
+												assetId={
+													media.mainMedia.videoId
+												}
+												duration={
+													media.mainMedia.duration
+												}
+												posterImage={
+													media.mainMedia.images
+												}
+												overrideImage={media.imageUrl}
+												width={media.mainMedia.width}
+												height={media.mainMedia.height}
+												origin={media.mainMedia.origin}
+												mediaTitle={headlineText}
+												expired={
+													media.mainMedia.expired
+												}
+												format={format}
+												isMainMedia={true}
+												hideCaption={true}
+												role="inline"
+												stickyVideos={false}
+												kickerText={kickerText}
+												pauseOffscreenVideo={
+													pauseOffscreenVideo
+												}
+												showTextOverlay={
+													containerType ===
+													'fixed/video'
+												}
+											/>
+										</Island>
+									</div>
+								) : (
+									<div>
+										<CardPicture
+											mainImage={
+												media.imageUrl
+													? media.imageUrl
+													: media.mainMedia.images.reduce(
+															(prev, current) =>
+																prev.width >
+																current.width
+																	? prev
+																	: current,
+													  ).url
```

Comment:
Here I am assuming the `mainImage` in the array of `posterImage` is the one with the largest width. Am I right to do so?

Reply:
Not sure I get what you're saying here. Merging for now and will ask you when you're back!